<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassGeneral</class>
 <widget class="QWidget" name="ClassGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassGeneral</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Documentation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="m_pClassL">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="m_pClassNameLE"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_pStereotypeL">
       <property name="text">
        <string>Stereotype</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_pStereotypeCBB">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_pPackageL">
       <property name="text">
        <string>Package Path</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KLineEdit" name="m_pPackageLE"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_pVisibilityL">
       <property name="text">
        <string>Visibility</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="m_pVisibilityCBB">
       <item>
        <property name="text">
         <string>Public</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Private</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Protected</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Implementation</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="m_pAbstractCK">
       <property name="text">
        <string>Abstract</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
